### PR TITLE
Add instructions for replacing an existing repository server

### PR DESCRIPTION
**Pull request by DrDaveD**

### CODE REVIEW

File: docs/data/external-oasis-repos.md [hunk 1]
```diff
@@ -191,12 +191,28 @@ Once the repository is fully replicated on the OSG, the VO may proceed in publis
...
 
-Changing the URL of a Repository on OASIS
------------------------------------------
+Replacing an Existing OASIS Repository Server
+---------------------------------------
 
-If necessary, it is possible to change the URL of the repository server; simply have the repository administrator
-open a support ticket with the new value, and OSG operations will update OIM's OASIS repository URL for the VO.
-The GOC Stratum-1 will then be updated within an hour.
+There are two recommended ways to replace the server for an existing ``*.opensciencegrid.org`` or ``*.osgstorage.org`` repository, one without GOC intervention and one with:
```
Comment (brianhlin) [inline docs/data/external-oasis-repos.md]: Why the use of double-backticks throughout this section?

Comment (DrDaveD) [reply docs/data/external-oasis-repos.md]: I thought I was just following prior practice in the file, but now I see that it was single backticks.  I will switch it.

**Comment by DrDaveD** [comment]
It's mostly about replacing the server with a new server, not replacing the contents.

Hopefully I'll be able to address the rest of your requested changes tomorrow.  Meanwhile the person that prompted me to write it has already done his replacement based on the submitted condition of my PR so it has been a low priority for me.

**Comment by DrDaveD** [comment]
Ok @brianhlin please re-review.

**Pull request merged**